### ISSUE TITLE
Upgrading to .net 7 fails

### ISSUE THREAD

**Issue by Lopah** [GeorgDangl/Nuke.GitHub#5]
I've tried upgrading this package to nuke 7.0.0 (https://github.com/Lopah/Nuke.GitHub/tree/feature/upgrade-to-nuke7) but when I try to a simple `nuke` command, i'm getting an error: 
```Unhandled exception. System.TypeInitializationException: The type initializer for 'Nuke.Common.NukeBuild' threw an exception.
 ---> System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
Could not load type 'Nuke.Common.Tooling.ISettingsEntity' from assembly 'Nuke.Common, Version=7.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   at System.Reflection.Assembly.GetTypes()
   at Nuke.Common.Host.<>c.<get_AvailableTypes>b__7_0(Assembly x) in /_/source/Nuke.Build/Host.Activation.cs:line 29
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToArray()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.OrderedEnumerable`1.GetEnumerator()+MoveNext()
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Enumerable.TryGetFirst[TSource](IEnumerable`1 source, Boolean& found)
   at System.Linq.Enumerable.First[TSource](IEnumerable`1 source)
   at Nuke.Common.Host.get_Default() in /_/source/Nuke.Build/Host.Activation.cs:line 20
   at Nuke.Common.NukeBuild..cctor() in /_/source/Nuke.Build/NukeBuild.Statics.cs:line 33
System.TypeLoadException: Could not load type 'Nuke.Common.Tooling.ISettingsEntity' from assembly 'Nuke.Common, Version=7.0.0.0, Culture=neutral, PublicKeyToken=null'.
   --- End of inner exception stack trace ---
   at Nuke.Common.NukeBuild.Execute[T](Expression`1[] defaultTargetExpressions) in /_/source/Nuke.Build/NukeBuild.cs:line 78
   at Build.Main() in C:\Repository\Nuke.GitHub\build\Build.cs:line 26
```
I've found that the problem is in Nuke.Common.Host.AvailableTypes -> it finds a type Nuke.Common.Tooling.ISettingsEntity which it can't find in Nuke.Common. I've run out of ideas where to look.

**Comment by Lopah** [comment]
Nevermind, I found the issue. It seems that it's next to impossible to get .build project to work when it's referencing Nuke.WebDocu, which references old version of nuke. This then results in the error above. How would you like to proceed? Should I open up a PR with broken .build project? It seems that both Nuke.WebDocu and Nuke.Github depend on each other, making a clean upgrade hard.

**Comment by GeorgDangl** [comment]
Yeah, this circular dependency is a bit of a problem😀 I’ll push an update later today, probably in a few hours. 

I think I’ll archive one of the repositories and merge the two projects into a single one.

**Comment by GeorgDangl** [comment]
I've just released v5.0.0, should be working now😀
